components/proxy/src/test/java/com/hotels/styx/startup/CoreMetricsTest.java

```diff
@@ -120,7 +120,20 @@ public void registersJvmMetrics() {
                 "jvm.thread.timed_waiting.count",
                 "jvm.thread.waiting.count",
                 "jvm.uptime",
-                "jvm.uptime.formatted"
-        ));
+                "jvm.uptime.formatted",
+
+// Unix system only
+//                "jvm.os.fileDescriptors.max",
+//                "jvm.os.fileDescriptors.open",
+
+                "jvm.os.cpu.process.load",
+                "jvm.os.cpu.process.time",
+                "jvm.os.cpu.system.load",
+                "jvm.os.memory.physical.free",
+                "jvm.os.memory.physical.total",
+                "jvm.os.memory.virtual.committed",
+                "jvm.os.swapSpace.free",
+                "jvm.os.swapSpace.total"
+                ));
```

Comment:
It feels odd that the operating system metrics are scoped under JVM metrics. I would consider moving them to the root level. Like so:

`os.swapSpace.free` or `os.memory.virtual.committed`.